model_compression_toolkit/qat/pytorch/quantizer/quantizer_utils.py

```diff
@@ -69,12 +69,47 @@ def fix_range_to_include_zero(range_min: torch.Tensor,
     return min_range_adj, max_range_adj
 
 
+def symmetric_quantizer(tensor_data: torch.Tensor,
+                        threshold: torch.Tensor,
+                        n_bits: int,
+                        sign: bool = False) -> torch.Tensor:
+    """
+    Quantize a tensor according to given range (min, max) and number of bits.
```

Comment:
actually, it's quantized according to number of bits and (a float) threshold. min-max are derived from the threshold (unlike uniform quantization)